Maple2.Server.Game/Manager/Config/BuffManager.cs

```diff
@@ -197,7 +203,7 @@ private bool TryAdd(Buff buff) {
     }
 
     public List<Buff> EnumerateBuffs() => buffs.Values.SelectMany(list => list).ToList();
-    public List<Buff> EnumerateBuffs(int buffId) => this.buffs.TryGetValue(buffId, out List<Buff>? buffs) ? buffs : [];
+    public List<Buff> EnumerateBuffs(int buffId) => this.buffs.TryGetValue(buffId, out List<Buff>? buffs) ? buffs.ToList() : [];
```

Comment:
isn't already a list?

Reply:
This creates another instance of a list. I do this because the List from the buff dict can be modified and can cause issues while iterating if the list is modified. 
TBH I might just change it from a List to another ConcurrentDictionary to avoid this later down the line.